src/components/App/App.js

```diff
@@ -196,6 +197,10 @@ const App = () => {
     <div className="app">
       <ToastContainer />
       <Modal title={modal.title}>{modal.content}</Modal>
+      <Helmet>
```

Comment:
I'm not sure I see the effect. What this code suppose to do?

Reply:
I suppose that is Redundant. I have removed it.